platform/jvm/src/jni.rs

```diff
@@ -1181,6 +1181,31 @@ pub extern "system" fn Java_io_bitdrift_capture_Jni_isRuntimeEnabled(
   .into()
 }
 
+#[no_mangle]
+pub extern "system" fn Java_io_bitdrift_capture_Jni_captureRuntimeIntValue(
```

Comment:
oh and you also have to add this method signature to https://github.com/bitdriftlabs/capture-sdk/blob/b2d27d34f7951772101eb7016b806f325dadcc07/platform/jvm/jni_symbols.lds